mne/utils/docs.py

```diff
@@ -3495,6 +3520,48 @@ def _reflow_param_docstring(docstring, has_first_line=True, width=75):
 For testing the lower tail (``tail=-1``), don't subtract ``pval`` from 1.
 """
 
+docdict['time_bandwitdh_tfr'] = """
+time_bandwidth : float
+    Time x (Full) Bandwidth product. Should be ``≥ 2.0``. The frequency
+    resolution achieved depends on ``n_cycles`` and ``time_bandwdith``. For a
+    given time-window length, a finer frequency resolution is achieved by
+    decreasing ``time_bandwidth`` at the cost of the number of good tapers. See
```

Comment:
I wouldn't frame it this way, you use more tapers to have higher frequency smoothing (either because you want to study higher frequencies where higher frequency smoothing is beneficial or you want to reduce noice by introducing more smoothing), so if you wanted to increase frequency resolution you likely wouldn't use multitapering.